src/components/change-log.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+class ChangeLog extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2 className='ds-h2'>Change Log</h2>
+        <p className='ds-text'>
+          The Change log is updated with each significant change to the API with the details that you need to be aware of.
+        </p>
+
+        <h3 className='ds-h3'>Submission API</h3>
+        <p className='ds-text'>
+          The Change log is updated with each significant change to the API with the details that you need to be aware of.
+        </p>
+
+        <ul>
+          <li>
+            <strong>8/15/20: Add "eligiblePopulationExclusion" validation for CQM measures</strong>
```

Comment:
Looks fine, but since this repo is already live, these dates look incorrect

Reply:
Let me confirm with Michelle

Reply:
Dates confirmed with Michelle

Reply:
When are we publishing this though?  Seems odd if we're publishing these changes before 8/15